stp_zmq/zstack.py

```diff
@@ -130,6 +130,7 @@ def __init__(self, name, ha, basedirpath, msgHandler, restricted=True,
         self._stashed_to_disconnected = {}
         self._stashed_pongs = set()
         self._received_pings = set()
+        self._waiting_messages = dict()
```

Comment:
1) It's better to use `{}` to init a new dict.
2) How do we limit the number of items in `_waiting_messages` to avoid OOM?